src/test/java/software/aws/neptune/gremlin/GremlinPreparedStatementTest.java

```diff
@@ -33,21 +31,21 @@ public class GremlinPreparedStatementTest extends GremlinStatementTestBase {
     private static final String HOSTNAME = "localhost";
     private static final int PORT = 8181; // Mock server uses 8181.
     private static final int MAX_CONTENT_LENGTH = 500000; // Took from PropertyGraphSerializationModule.
-    private NeptunePreparedStatementTestHelper neptunePreparedStatementTestHelper;
+    private static NeptunePreparedStatementTestHelper neptunePreparedStatementTestHelper;
 
-    @BeforeEach
-    void initialize() throws SQLException, IOException, InterruptedException {
-        MockGremlinDatabase.startGraph();
+    @BeforeAll
```

Comment:
This is different from how to worked before. BeforeAll is fine if we don't expect any changes to data. Do you expect any mutating queries to be added to this test file?

Reply:
There is no mutating queries for this test, I was testing to see if before/after all affected testing, but it's not for any meaningful changes. I will revert.